backend/app/helpers/answer_helpers.rb

```diff
@@ -0,0 +1,9 @@
+module AnswerHelpers
```

Comment:
talking with a friend, he suggested to use concern instead of helper in this situation

Reply:
Ok Cool I will see what I can do

Reply:
https://blog.appsignal.com/2020/09/16/rails-concers-to-concern-or-not-to-concern.html